test/knexfile.js

```diff
@@ -117,6 +117,18 @@ var testConfigs = {
     seeds: seeds
   },
 
+  redshift: {
+    dialect: 'redshift',
+    connection: testConfig.redshift || {
+      adapter:  "postgresql",
+      database: "knex_test",
+      user:     "knex_test"
```

Comment:
Shouldn't the user here be `postgres`, if it's using the same user as the postgres configuration?